### PR TITLE
Update NPM Dependencies

### PR DESCRIPTION
**Pull request by fuhrmeistery**
- Update Dependencies For All Hooks
- Update NPM Dependencies For All Scanners
- Update Root Packages
- Replaces PR #547

All packages were upgraded using the `npm-check-updates` package (`ncu -u && npm i`)

### CODE REVIEW

File: hooks/cascading-scans/package.json [hunk 1]
```diff
@@ -37,15 +37,15 @@
...
     "matcher": "^4.0.0",
     "mustache": "^4.2.0"
   },
   "devDependencies": {
-    "@types/lodash": "^4.14.169",
-    "@types/node": "^14.0.14",
-    "jest": "^27.0.3",
-    "typescript": "^3.9.5"
+    "@types/lodash": "^4.14.171",
+    "@types/node": "^16.3.3",
```
Comment (twwd) [inline hooks/cascading-scans/package.json:47]: We are using node LTS and thus the types should be on latest 14.X

File: hooks/notification/package.json [hunk 2]
```diff
@@ -23,23 +23,23 @@
...
-    "@types/jest": "^26.0.20",
-    "@types/lodash": "^4.14.168",
-    "@types/mustache": "^4.1.1",
-    "@types/node": "^14.14.33",
-    "@types/nodemailer": "^6.4.1",
-    "jest": "^27.0.3",
+    "@types/jest": "^26.0.24",
+    "@types/lodash": "^4.14.171",
+    "@types/mustache": "^4.1.2",
+    "@types/node": "^16.3.3",
```
Comment (twwd) [inline hooks/notification/package.json:29]: See above

File: package.json [hunk 3]
```diff
@@ -37,18 +37,18 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
-    "@types/jest": "^26.0.23",
-    "@types/node": "^14.14.32",
-    "eslint": "^7.14.0",
+    "@types/jest": "^26.0.24",
+    "@types/node": "^16.3.3",
```
Comment (twwd) [inline package.json:41]: See above

**Review by twwd — CHANGES_REQUESTED** [review]

File: hooks/cascading-scans/package.json (hunk 1 above)
Comment (fuhrmeistery) [reply hooks/cascading-scans/package.json]: Good Catch

File: hooks/cascading-scans/package.json [hunk 4]
```diff
@@ -1,51 +1 @@
...
-    "mustache": "^4.2.0"
-  },
-  "devDependencies": {
-    "@types/lodash": "^4.14.171",
-    "@types/node": "^16.3.3",
-    "jest": "^27.0.6",
-    "typescript": "^4.3.5"
-  }
-}
+{"name":"@securecodebox/hook-cascading-scans","version":"1.0.0","description":"secureCodeBox Hook to cascade scan in an declarative manner.","homepage":"https://www.secureCodeBox.io","repository":{"type":"git","url":"git+https://github.com/secureCodeBox/secureCodeBox.git"},"main":"hook.js","scripts":{"build":"npx typescript hook.ts --sourceMap","test":"jest . --verbose false"},"keywords":["secureCodeBox","security","hook"],"author":{"name":"iteratec GmbH","email":"securecodebox@iteratec.com","url":"https://www.iteratec.com"},"contributors":[{"name":"Jannik Hollenbach","url":"https://github.com/J12934"},{"name":"Robert Seedorff","url":"https://github.com/rseedorff"}],"bugs":{"url":"https://github.com/secureCodeBox/secureCodeBox/issues"},"license":"Apache-2.0","dependencies":{"@kubernetes/client-node":"^0.15.0","lodash":"^4.17.21","matcher":"^4.0.0","mustache":"^4.2.0"},"devDependencies":{"@types/lodash":"^4.14.171","@types/node":"^14.17.5","jest":"^27.0.6","typescript":"^4.3.5"}}
```
Comment (twwd) [inline hooks/cascading-scans/package.json:1]: I think it was not intended that all line breaks are gone in this file 😅

File: hooks/notification/package.json [hunk 5]
```diff
@@ -1,45 +1 @@
...
-    "@types/nunjucks": "^3.1.5",
-    "axios": "^0.21.1",
-    "js-yaml": "^4.1.0",
-    "lodash": "^4.17.21",
-    "nodemailer": "^6.6.3",
-    "nunjucks": "^3.2.3",
-    "typescript": "^4.3.5"
-  }
-}
+{"name":"@securecodebox/hook-notification","version":"1.0.0","description":"secureCodeBox Hook for Notification","main":"hook.js","scripts":{"test":"jest","build":"npx tsc hook.ts --sourceMap"},"repository":{"type":"git","url":"git+https://github.com/secureCodeBox/secureCodeBox.git"},"keywords":["secureCodeBox","security","hook"],"author":"iteratec GmbH","license":"Apache-2.0","bugs":{"url":"https://github.com/secureCodeBox/secureCodeBox/issues"},"homepage":"https://github.com/secureCodeBox/secureCodeBox#readme","devDependencies":{"@types/jest":"^26.0.24","@types/lodash":"^4.14.171","@types/mustache":"^4.1.2","@types/node":"^14.17.5","@types/nodemailer":"^6.4.4","jest":"^27.0.6","ts-jest":"^27.0.3"},"dependencies":{"@kubernetes/client-node":"^0.15.0","@types/js-yaml":"^4.0.2","@types/nunjucks":"^3.1.5","axios":"^0.21.1","js-yaml":"^4.1.0","lodash":"^4.17.21","nodemailer":"^6.6.3","nunjucks":"^3.2.3","typescript":"^4.3.5"}}
```
Comment (twwd) [inline hooks/notification/package.json:1]: See above

**Review by twwd — CHANGES_REQUESTED** [review]

File: hooks/cascading-scans/package.json (hunk 4 above)
Comment (fuhrmeistery) [reply hooks/cascading-scans/package.json:1]: I really do hate NPM

**Review by twwd — APPROVED** [review]

**Pull request merged**